wordle-web/src/components/NavigationBar.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-toolbar>
+    <template v-slot:prepend>
+      <v-btn icon to="/">
+        <v-icon color="deep-orange-accent-3" size="x-large" icon="mdi-check-circle"></v-icon>
```

Comment:
I'm guessing this was going to be worked on later or my understanding is incorrect, but requirement 2-2 asks for the icon to be from somewhere on the web.  I might be wrong and you can feel free to ignore this, but I'm assuming they want us to pull the image from somewhere else to get experience with importing svg files.